tools/docker/Dockerfile.ubuntu18.04

```diff
@@ -74,7 +74,7 @@ RUN git clone -b sgx_2.11_for_occlum https://github.com/occlum/linux-sgx && \
 
 # Install Rust
 ENV PATH="/root/.cargo/bin:$PATH"
```

Comment:
In the updates to Git submodule, the branch `sgx_2.13_for_occlum` of Rust SGX SDK is selected. But in the Dockerfile, the branch of Intel SGX SDK is `sgx_2.11_for_occlum`. Is this inconsistency intentional or should be fixed? There is no guarantee that Rust SGX SDK 2.x is compatible with Intel SGX SDK 2.y, if x is not equal to y.

Reply:
I haven't update the dockerfile, since the dockerfile is meaningless now, I think we should update dockerfile according to latest occlum.

ps. The temporary docker image is created by the dockerfile of occlum 0.23.7 (with clang installed)